app/src/main/res/layout/activity_splash.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout
```

Comment:
I dont think you need to use coordinator layout here. You can just use the relative layout.